systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh

```diff
@@ -6,10 +6,64 @@ KEYCLOAK_INSTANCE_NAMESPACE=$3
 
 SCRIPT_PATH=$(dirname "${BASH_SOURCE[0]}")
 
+function create_keycloak_cr() {
+cat <<EOF > /tmp/keycloak_pg_example.yaml
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: example-keycloak
+  labels:
+    app: sso
+spec:
+  instances: 1
+  externalAccess:
+    enabled: True
```

Comment:
This will probably cause issues on minikube, did you try it?

Reply:
PR job (minikube) passed. AZP pipeline is running as well. Seems OK.

Reply:
Should be no problem. It was always used in example deployments we used. 15.0.4 etc.
https://raw.githubusercontent.com/keycloak/keycloak-operator/11.0.1/deploy/examples/keycloak/keycloak.yaml